packages/docprovider/src/component.tsx

```diff
@@ -20,6 +20,7 @@ type Props = {
   contentType: string;
   format: string;
 };
+const DOCUMENT_TIMELINE_URL = 'api/collaboration/timeline';
```

Comment:
We already have it [here](https://github.com/jupyterlab/jupyter-collaboration/blob/3126a81b78174cbc32c5f05d4678458f2d01f3eb/packages/docprovider/src/requests.ts#L15) and [here](https://github.com/jupyterlab/jupyter-collaboration/blob/3126a81b78174cbc32c5f05d4678458f2d01f3eb/packages/docprovider-extension/src/filebrowser.ts#L46). Should it be in one place?

Reply:
Yes I will fix that. Thanks for the feedback !